### PR TITLE
Feature/orca 622 Research Adding big int support to graphql

### PR DESCRIPTION
**Pull request by AndrewDorn**
## Summary of Changes

Added bigint support to GraphQL.

Addresses [ORCA-622: Research Adding big int support to graphql](https://bugs.earthdata.nasa.gov/browse/ORCA-622)

## Changes

* Added int8 to graphql

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Run against AWS deployment.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] CHANGELOG.md
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
    - [x] Unit tests
- [x] New and existing unit tests pass locally with my changes
    - [x] Automated tests passing (if not fork)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

File: modules/graphql_1/main.tf [hunk 1]
```diff
@@ -243,15 +243,15 @@ DEFINITION
...
+  description = "Allow inbound communication from LB on container port. Allow outbound communication to get image."
   vpc_id      = var.vpc_id
 
   ingress {
     description      = "Container port communication."
     from_port        = local.graphql_port
     to_port          = local.graphql_port
     protocol         = "tcp"
-    cidr_blocks      = [data.aws_vpc.primary.cidr_block]
+    security_groups  = [aws_security_group.gql_lb_security_group.id]
```
Comment (AndrewDorn) [inline modules/graphql_1/main.tf:254]: Found I can make this more restrictive during GQL security research.

**Review by rizbihassan — CHANGES_REQUESTED** [review]
looks good.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ and includes an additional section for migration notes.
 
 ## [Unreleased]
 ### Added
-- *ORCA-554*, *ORCA-561*, *ORCA-579*, *ORCA-581*
+- *ORCA-554*, *ORCA-561*, *ORCA-579*, *ORCA-581*, *ORCA-622*
   - GraphQL image, service, and Load Balancer will now be deployed by TF.
   - *ORCA-557* Added `orca_graphql_load_balancer_dns_name` to output variables for GraphQL integration.
   - *ORCA-420* Added Internal Reconcile Report Mismatch functionality to GraphQL.
```
[the remark this answers is not in the document]
Comment (AndrewDorn) [reply CHANGELOG.md:21]: Added

File: modules/graphql_1/main.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (AndrewDorn) [reply modules/graphql_1/main.tf:254]: Adding it to this `egress` block would prevent the policy from accessing ghcr. If you're referring to a different `egress` block, that should be a separate card to allow for testing.

**Pull request merged**